components/auth0.key.manager/src/main/java/org/wso2/auth0/client/Auth0ConnectorConfiguration.java

```diff
@@ -0,0 +1,106 @@
+ * under the License.
+ */
+
+package org.wso2.auth0.client;
+
+import org.osgi.service.component.annotations.Component;
+import org.wso2.carbon.apimgt.api.model.ConfigurationDto;
+import org.wso2.carbon.apimgt.api.model.KeyManagerConnectorConfiguration;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@Component(
+        name = "auth0.configuration.component",
+        immediate = true,
+        service = KeyManagerConnectorConfiguration.class
+)
+public class Auth0ConnectorConfiguration implements KeyManagerConnectorConfiguration {
+    @Override
+    public String getImplementation() {
+        return Auth0OAuthClient.class.getName();
+    }
+
+    @Override
+    public String getJWTValidator() {
+        return null;
+    }
+
+    @Override
+    public List<ConfigurationDto> getConnectionConfigurations() {
+        List<ConfigurationDto> configurationDtoList = new ArrayList<>();
+        configurationDtoList
+                .add(new ConfigurationDto(Auth0Constants.CLIENT_ID, "Client ID", "input",
+                        "Client ID of service Application", "",
+                        true,
+                        false, Collections.emptyList(), false));
+        configurationDtoList
+                .add(new ConfigurationDto(Auth0Constants.CLIENT_SECRET, "Client Secret", "input",
+                        "Client Secret of service Application", "", true,
+                        true, Collections.emptyList(), false));
+        configurationDtoList
+                .add(new ConfigurationDto(Auth0Constants.AUDIENCE, "Audience", "input",
+                        "Audience of the Admin API", "https://[tenant].[region].auth0.com/api/v2/",
+                        true,
+                        false, Collections.emptyList(), false));
+        return configurationDtoList;
+    }
+
+    @Override
+    public List<ConfigurationDto> getApplicationConfigurations() {
+        List<ConfigurationDto> configurationDtoList = new ArrayList<>();
+        configurationDtoList
+                .add(new ConfigurationDto("app_type", "Application Type", "select",
+                        "Type Of Application to create", "regular_web", false,
+                        false, Arrays.asList("regular_web", "native", "spa", "non_interactive"), false));
+        configurationDtoList
+                .add(new ConfigurationDto("token_endpoint_auth_method",
+                        "Token Endpoint Authentication Method", "select",
+                        "How to Authenticate Token Endpoint", "client_secret_basic", true,
+                        true, Arrays.asList("client_secret_basic", "client_secret_post"),
+                        false));
+        configurationDtoList
+                .add(new ConfigurationDto("audience_of_api",
+                        "Audience of the API", "text",
+                        "Audience of the API which intended to use this application", "",
```

Comment:
The audience of the API which intends to use the application.

Reply:
fixed